modin/pandas/test/utils.py

```diff
@@ -630,8 +630,14 @@ def eval_general(
     comparator=df_equals,
     __inplace__=False,
     check_exception_type=True,
+    nonacceptable_exception_types=None,
+    check_kwargs_callable=True,
```

Comment:
It's unused?

Reply:
Yes for now (`check_kwargs_callable` parameter is removed).